storage/client/queries.go

```diff
@@ -410,9 +410,9 @@ func (qf QueryFactory) RuntimeTransactionsQuery() string {
 				-- Otherwise, every tx will be returned as many times as there are related addresses for it. 
 				AND $3::text IS NOT NULL
 			WHERE (txs.runtime = '%[2]s') AND
-						($1 IS NULL OR txs.round = $1::bigint) AND
-						($2 IS NULL OR txs.tx_hash = $2::text) AND
-						($3 IS NULL OR rel_accounts.account_address = $3::text)
+						($1::bigint IS NULL OR txs.round = $1::bigint) AND
+						($2::text IS NULL OR txs.tx_hash = $2::text OR txs.tx_eth_hash = $2::text) AND
+						($3::text IS NULL OR rel_accounts.account_address = $3::text)
```

Comment:
Without the typecasts, sql complains with `"msg": "storage error: ERROR: could not determine data type of parameter $1 (SQLSTATE 42P08)"`

Reply:
TIL. Thanks.